watertap/unit_models/tests/test_cstr.py

```diff
@@ -456,16 +457,18 @@ def test_costing(self):
         m.fs.unit.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
         m.fs.costing.cost_process()
         m.fs.costing.add_LCOW(m.fs.unit.control_volume.properties_out[0].flow_vol)
+        m.fs.costing.initialize()
+        m.objective = Objective(expr=m.fs.costing.LCOW)
         assert_units_consistent(m)
-        results = solver.solve(m)
+        results = solver.solve(m, tee=True)
 
         assert_optimal_termination(results)
 
         # Check solutions
         assert pytest.approx(526.45 * 2, rel=1e-5) == value(
             m.fs.unit.costing.capital_cost
         )
-        assert pytest.approx(1.07948e-5, rel=1e-5) == value(m.fs.costing.LCOW)
+        assert pytest.approx(8.94365e-07, rel=1e-5) == value(m.fs.costing.LCOW)
```

Comment:
@MarcusHolly can you have a look at this? It seems like this flowsheet had degrees of freedom but no objective, and therefore the LCOW could converge to an arbitrary value.

I've fixed it here to what I got out of the optimization.

Reply:
Sure, I can take a look at it soon, just not right now...